src/components/Header/Navbar.tsx

```diff
@@ -97,6 +105,10 @@ const PrimaryLink = styled(Link)`
   color: ${white};
 `;
 
+const PrimaryA = styled.a`
+  color: ${white};
```

Comment:
[nit] Let's define a shared `primaryStyle` using the `css` prop as is done for secondary links